src/zc/Zcmb_footer.adoc

```diff
@@ -0,0 +1,12 @@
+
```

Comment:
do we still need the footer?

Reply:
I've removed all of the footers from the consolidated chapter.